File: src/main/java/com/ject/studytrip/trip/domain/repository/TripReportStudyLogRepository.java
package com.ject.studytrip.trip.domain.repository;

import com.ject.studytrip.trip.domain.model.TripReportStudyLog;
import java.util.List;

public interface TripReportStudyLogRepository {
    void saveAll(List<TripReportStudyLog> tripReportStudyLogs);
}

File: src/main/kotlin/com/ject/studytrip/trip/domain/repository/TripReportStudyLogRepository.kt
package com.ject.studytrip.trip.domain.repository

import com.ject.studytrip.trip.domain.model.TripReportStudyLog

interface TripReportStudyLogRepository {
    fun saveAll(tripReportStudyLogs: List<TripReportStudyLog>)
}
